data/campaigns/Heir_To_The_Throne/scenarios/42_A_Crisis_of_Leadership.cfg

```diff
@@ -0,0 +1,1378 @@
+        [/message]
+    [/event]
+    [event]
+        name=last breath
+        {FILTER id=leader3}
+        [message]
+            speaker=leader3
+            message=_"Filthy pinkskins!"
+        [/message]
+    [/event]
+    [event]
+        name=last breath
+        {FILTER id=leader4}
+        [message]
+            speaker=leader4
+            message=_"Curse you all!"
+        [/message]
+    [/event]
+    [event]
+        name=last breath
+        {FILTER id=leader6}
+        [message]
+            speaker=leader6
+            message=_"Gah!!"
+        [/message]
+    [/event]
+
+    #############################
+    # BAZUR DIES
+    #############################
+    [event]
+        name=last breath
+        {FILTER id=Khargol}
+
+        [message]
+            speaker=Khargol
+            #po: Konrad may not have struck the finishing blow, but it'll be either him or one of his soldiers
+            message=_"Impossible! Arr, how shameful to die at the hands of some stupid assassins..."
+        [/message]
+        {IF} {VARIABLE_CONDITIONAL s42_failed_achievement not_equals yes} {THEN(
+            {ACHIEVE s42}
+        )} {/IF}
+        {CLEAR_VARIABLE s42_failed_achievement}
+
+        [event]
+            name=die
+            {IF} {NOT({HAVE_UNIT id=leader3})} {THEN(
+                {NAMED_UNIT 3 "Orcish Warrior" 24 1 leader3 _"Gado-har" canrecruit=yes} {FACING se}
+            )} {/IF}
+            {REVEAL( radius=1 {FILTER id=leader3} )}
+            [message]
+                speaker=leader3
+                message=_"Khargol’s dead! The assassins killed our chief!"
+            [/message]
+
+            #------------------------
+            # FAGH-MOK ESCAPES HIS CAGE
+            #------------------------
+            [message]
+                speaker=faghmok
+                image_pos=right
+                mirror=yes
+                message=_"Now is my chance! Set me free, princess, so I can take command of the horde!"
+            [/message]
+            [message]
+                speaker=Lisar {LISAR_VARIATION mad}
+                message=_"Set you free? Nice try. You’re not getting out of that cage until Konrad returns."
+            [/message]
+            [message]
+                speaker=faghmok
+                image_pos=right
+                mirror=yes
+                message=_"Stupid girl, you think the other orcs are just going to let him walk out? I must take command. Set me free or your friends all die!"
+            [/message]
+            [message]
+                speaker=Lisar {LISAR_VARIATION mad}
+                message=_"Rrr... insufferable. I don’t know how mother stands working with orcs."
+            [/message]
+            [message]
+                speaker=Lisar {LISAR_VARIATION mad}
+                message=_"Go. But don’t dare betray us, or I swear to personally find you and kill you. If you know anything about me, you’ll know that’s a promise I can keep."
```

Comment:
don't you dare

Reply:
I may not have the right terminology, but I think the issue that I have is there's no subject. Who or what is doing the betraying? We can infer that from the context, but when it's missing it sounds (to me) informal, even uneducated.